File: mobile/src/main/java/my/work/retailerautoapp/RetailerAutoAppService.java
package my.work.retailerautoapp;


import com.google.android.apps.auto.sdk.CarActivity;
import com.google.android.apps.auto.sdk.CarActivityService;

public class RetailerAutoAppService extends CarActivityService {

    @Override
    public Class<? extends CarActivity> getCarActivity() {
        return  RetailerAutoActivity.class;
    }

}

File: mobile/src/main/java/my/work/retailerautoapp/RetailerAutoAppService.kt
package my.work.retailerautoapp

import com.google.android.apps.auto.sdk.CarActivity
import com.google.android.apps.auto.sdk.CarActivityService

class RetailerAutoAppService : CarActivityService() {
    override fun getCarActivity(): Class<out CarActivity> = RetailerAutoActivity::class.java
}